index.html

```diff
@@ -240,15 +240,30 @@ <h1>Biometrics</h1>
+    <p>Although companies and industry groups <a href="https://www.technewsworld.com/story/86512.html">boast</a> of the technology’s accuracy, facial recognition systems do not yet seem to be accurate enough to consistently identify individuals across various demographic groups. Even Detroit’s Police Chief has said that his department’s facial recognition software is inaccurate<a href="https://www.vice.com/amp/en_us/article/dyzykz/detroit-police-chief-facial-recognition-software-misidentifies-96-of-the-time"> 96% of the time</a>.</p>
+    <p>CPD’s Latent Print Unit (LPU), which examines fingerprints from crime scenes, has come under a great deal of <a href="https://www.bettergov.org/news/bad-police-fingerprint-work-undermines-chicago-property-crime-cases">scrutiny</a> for having opaque, ineffective, and outdated practices. An external review found that LPU examiners lack adequate training, and that the unit has not kept up with reformed standards in the field. Thus, the Cook County Public Defender’s Office usually <a href="https://www.documentcloud.org/documents/5765026-Henderson-Admissibility-Motions.html">motions</a> for courts to ignore testimonies from CPD’s fingerprint analysts.</p>
     <h3> <span class="glyphicon glyphicon-exclamation-sign"></span> Legal and Privacy Concerns</h3>
-    <p> The secure storage of biometric data is increasingly important as biometrics such as fingerprints are used for authentication purposes. In Illinois we have the <a href="http://www.ilga.gov/legislation/ilcs/ilcs3.asp?ActID=3004&ChapterID=57">Biometric Information Privacy Act (BIPA)</a> which regulates the collection of biometric data, by required consent as well as limits on how long the data can be stored. There are currently <a href="http://www.biometricupdate.com/201508/states-considering-biometrics-capture-laws-may-look-to-illinois-privacy-laws">lawsuits</a> against companies like Facebook and Shutterfly for collecing biometric information for commercial purposes in violation of this act. Unfortunately, law enforcement may not be regulated by BIPA. </p>
+    <p>Illinois' Biometric Information Protection Act (BIPA), passed in 2008, is one of the strongest biometric privacy laws in the country, requiring companies to obtain written consent from an individual to collect their biometric data, and also to store and dispose of that data securely. In 2019 LPL and our partner organizations helped win a supreme court case to <a href="https://lucyparsonslabs.com/posts/BIPA-Victory/">protect</a> BIPA. While private companies like Clearview and Facebook, which was recently <a href="https://www.courthousenews.com/judge-approves-650-million-settlement-in-facebook-biometric-case/">sued</a> for $650 million, are subject to BIPA, public agencies such as the police are not.</p>
```

Comment:
nit, `In 2019,`. 

Also Facebook was not sued for 650 recently, it _settled_ for 650 million this week.